jobbergate-agent/jobbergate_agent/jobbergate/submit.py

```diff
@@ -206,10 +207,16 @@ async def _reject_handler(params: DoExceptParams):
         preexec_fn = run_as_user(username)
 
     submit_dir = pending_job_submission.execution_directory or SETTINGS.DEFAULT_SLURM_WORK_DIR
+    async with handle_errors_async(
+        "The submission directory must exist and be an absolute path",
+        raise_exc_class=JobSubmissionError,
+        do_except=_reject_handler,
```

Comment:
`buzz. require_conditon` is more indicated for this check, but `do_except=_reject_handler` is just available on `handle_errors_async`, so I think we can keep it in this way for now.

Reply:
In this case, it would probably be better to do away with the Buzz stuff entirely and just use a vanilla if-then and call the `mark_as_rejected` locally instead o fusing the `_reject_handler`.

Reply:
solved in [6ba95c0](https://github.com/omnivector-solutions/jobbergate/pull/504/commits/6ba95c0ab4b60151f2eb70c9fa71b97f3a466a07)